### DISCUSSION TITLE
Facing issues in joining official discord server.

### DISCUSSION THREAD

**Discussion by J4e6eR** [tinygrad/tinygrad#2276] [General]
@geohotstan @wozeparrot How to join the discord server, I found the invite link from the tinygrad website but that doesn't seem to work for me. 
I don't know why I am getting this error, is there any alternative way to join ?? 
I've been trying to join since a month.
Helping me out in joining tinygrad's official discord server would truly be appreciable.
![image](https://github.com/tinygrad/tinygrad/assets/91129126/f007e935-0242-438a-bb67-22a2e983aa7f)

**Comment by wozeparrot** [comment]
Pretty sure you got banned.

**Reply by J4e6eR** [reply]
I don't know, I just messaged once in general channel and it wasn't even anything offensive, and I guess it followed the rules as well. 
I honestly don't know what happened. 
Can you please make some arrangements so that I can join again ?? @wozeparrot

**Comment by geohotstan** [comment]
@J4e6eR I looked at your previous PR and I think you actually did get banned.
I don't think you'll be unbanned either.
The reason being you're using up contributor's time and not providing anything in return
For instance: posting in discussions asking to be unbanned is wasting people's time since they'll have to read and close your post

Sorry :(

**Reply by J4e6eR** [reply]
@geohotstan I appreciate your quick response & thanks for wasting time on me by doing research related to my previous PR. Speaking about it, I guess it was helpful as it helped me and I guess would have helped beginners to get to know the flow of the code in this repo. 
Are you sure that I am banned or it's an assumption from your side??
And are you sure the real reason's being me wasting time or it's again an assumption from your side?? Because I've been advised by a contributor to create discussion and I guess discussions are meant to be discussed and do not intend to waste time. Moreover, it seems that there are lot of assumptions from your side!!

If you've done this deep research on my GitHub, I assume you might have my discord ID, if yes then we would have to report this as a vulnerability to GitHub and if no, then isn't it another assumption from your side??

Lastly, I don't aim to trigger any sort of argument between us, and I am sorry if any of the above sounded rude. Bro, we are on the same boat trying to help this repo and make a better AI for future.

**Reply by geohotstan** [reply]
:/
sail on bother
